### DISCUSSION TITLE
Welcome to java2uml Discussions!

### DISCUSSION THREAD

**Discussion by leluque** [leluque/java2uml#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by IagoFreitas05** [comment]
Hello! My name is Iago, i am a software developer since 2018 and java developer since 2022. I would like to contribute building new features, but first i want to understand about the architecture of the project.
